saleor/settings.py

```diff
@@ -589,9 +589,9 @@ def get_host():
         "task": "saleor.csv.tasks.delete_old_export_files",
         "schedule": crontab(hour=1, minute=0),
     },
-    "handle-sale-toggle": {
-        "task": "saleor.discount.tasks.handle_sale_toggle",
-        "schedule": initiated_sale_webhook_schedule,
+    "handle-promotion-toggle": {
```

Comment:
I am not convinced that we should replace the sale-toggle in the current way. 
I would expect that sale webhooks will be deprecated and removed in 4.0. I would also expect that all of my apps subscribed to the sale's webhooks will be receiving the events in the same form as previously. In that case, I would have time till 4.0 to apply changes in my apps and add implementation to support promotion events.
Stoping sending the Sale events sounds like a not intended breaking change 🤔

Reply:
Fixed